homeworks/09-js-advanced-2/script.js

```diff
@@ -0,0 +1,108 @@
+class Serializable {
+  serialize() {
+    const props = Object.getOwnPropertyNames(this);
```

Comment:
Using this method all values are retrieved as strings!
Mb Object.values match better...